hw/ip/keymgr/doc/_index.md

```diff
@@ -119,53 +124,49 @@ Once the `OwnerRootKey` is created, software can request key manager to advance
 
 When advance and generate calls are invoked from this state, the outputs and keys are indiscriminately updated with randomly computed values.
 Key manager enters disabled state based on direct invocation by software:
 * Advance from `OwnerRootKey`
 * Disable operation
 
 ### Invalid
-`Invalid` state is entered whenever key manager is disabled through the [life cycle connection](#life-cycle-connection) or when an operation encounters a [fault](#faults-and-operational-faults) .
-Upon `Invalid` entry, both the working state and the sideload keys are wiped with entropy directly.
-Note, this is different from `Disabled` state entry, which updates internal key with KMAC outputs but leaves sideload and software keys intact.
+`Invalid` state is entered whenever key manager is deactivated through the [life cycle connection](#life-cycle-connection) or when an operation encounters a [fault](#faults-and-operational-faults) .
+Upon `Invalid` entry, the internal key, the sideload key slots and the software keys are all wiped with entropy directly.
 
 #### Invalid Entry Wiping
-Since the life cycle controller can disable the key manager at any time, the key manager attempts to gracefully handle the wiping process.
-When the disable is seen, the key manager immediately begins wiping all keys (internal key, hardware sideload key, software key) with entropy.
-However, if an operation was already ongoing, the key manager waits for the transaction to complete gracefully before transitioning to invalid state.
+Since the life cycle controller can deactivate the key manager at any time, the key manager attempts to gracefully handle the wiping process.
+When the deactivated, the key manager immediately begins wiping all keys (internal key, hardware sideload key, software key) with entropy.
+However, if an operation was already ongoing, the key manager waits for the operation to complete gracefully before transitioning to invalid state.
 
-While waiting for the transaction to complete, the key manager continuously wipes all keys with entropy.
+While waiting for the operation to complete, the key manager continuously wipes all keys with entropy.
 
 ### Invalid and Disabled State
 
-Note that `Invalid` and `Disabled` states are functionally equivalent.
-The main difference between the two is "how" the state was reached.
+`Invalid` and `Disabled` states are functionally very similar.
+The main difference between the two is "how" the states were reached and the entry behavior.
 
-`Disabled` state is reached through intentional software commands.
-While `Invalid` state is reached through life cycle disable or operational faults.
+`Disabled` state is reached through intentional software commands where the internal key, sideload key slots and software key are not wiped.
+While `Invalid` state is reached through life cycle deactivation or operational faults where the internal key, sideload key slots and software key are wiped.
```

Comment:
I would change these lines (155/156) so that 156 is connected to 155 by a comma.  Basically replace the period from the end of 155 with a comma, and decapitalize the "While" in 156.  Otherwise 156 is a sentence fragment.

Reply:
ah yes good catch.